src/server_manager/web_app/app.ts

```diff
@@ -1254,6 +1255,9 @@ export class App {
     this.appRoot.setLanguage(languageCode, languageDir);
     document.documentElement.setAttribute('dir', languageDir);
     window.localStorage.setItem('overrideLanguage', languageCode);
+    this.appRoot.addEventListener(
```

Comment:
Every time you call setAppLAnguage you will add a new Event Listener.
Add that in the constructor instead.

Reply:
Done in 79fb1ea